GrayZone/machinetag.json

```diff
@@ -1,51 +1,51 @@
 {
   "namespace": "GrayZone",
   "description": "Gray Zone of Active defense includes all elements which lay between reactive defense elements and offensive operations. It does fill the gray spot between them. Taxo may be used for active defense planning or modeling.",
-  "version": 2,
+  "version": 3,
```

Comment:
We will be releasing version 3 somewhere next year, it will be maybe better to number this verion as 2.1. Thank You

Reply:
Thanks a lot for the feedback. The only issue, it should be an integer. I can also put the latest date of update if this is fine for you.

Reply:
Yep, the date will be just fine.
Thank You